wasm_samples/executor_custom_data_model/src/multisig.rs

```diff
@@ -1,24 +1,32 @@
-//! Arguments to register and manage multisig account
+//! Arguments attached on executing triggers for multisig accounts or transactions
 
-use alloc::{collections::btree_set::BTreeSet, vec::Vec};
+use alloc::{collections::btree_map::BTreeMap, vec::Vec};
 
-use iroha_data_model::{account::NewAccount, prelude::*};
+use iroha_data_model::prelude::*;
 use serde::{Deserialize, Serialize};
 
-/// Arguments to multisig account register trigger
+/// Arguments to register multisig account
 #[derive(Serialize, Deserialize)]
-pub struct MultisigRegisterArgs {
-    // Account id of multisig account should be manually checked to not have corresponding private key (or having master key is ok)
-    pub account: NewAccount,
-    // List of accounts responsible for handling multisig account
-    pub signatories: BTreeSet<AccountId>,
+pub struct MultisigAccountArgs {
+    /// Multisig account to be registered
+    /// WARNING: any corresponding private key allows the owner to manipulate this account as a ordinary personal account
+    pub account: PublicKey,
+    /// List of accounts and their relative weights of responsibility for the multisig
+    pub signatories: BTreeMap<AccountId, u8>,
+    /// Threshold of total weight at which the multisig is considered authenticated
+    pub quorum: u16,
+    /// Multisig transaction time-to-live based on block timestamps. Defaults to [`DEFAULT_MULTISIG_TTL_SECS`]
+    pub transaction_ttl_secs: Option<u32>,
```

Comment:
```suggestion
    #[serde(default = DEFAULT_MULTISIG_TTL_SECS)]
    pub transaction_ttl_ms: u32,
```

1. can we do it like this?
2. we use `ms` in `transaction.rs` as unix timestamp

I think there should always be a TTL defined. User an just put a huge number

Reply:
Ok, we can accept seconds as an user input and convert it to milliseconds here